bigquery/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+## 1.2.0
+
+### Implementation Changes
+- switch list_partitions helper to a direct metatable read (#5273)
```

Comment:
Can we normalize capitalization here, as well as wrapping code names (e.g., list_partitions) in backticks so that they render as code?